### PR TITLE
Add a "my_services" command

### PR DESCRIPTION
**Pull request by harrybiddle**
I love BurgerBot!

It would be really useful to be able to list which services you are subscribed to. I've added a command "my_services" that does this. **This is untested**! I don't know how one goes about testing a telegram bot command...

~I've also snuck in another change in this PR. When I used the bot I was initially dismayed that the service I was interested in wasn't listed by the "services" command. Eventually I realised that you can add pretty much any service! This wasn't clear to me initially. So, I have altered the help text to make it clearer that the list is non-exhaustive.~ *Edit* it seems like maybe this isn't the case? Not sure. Removed this change anyway...

### CODE REVIEW

File: burgerbot.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def __services(self, update: Update, _: CallbackContext) -> None:
     services_text = ""
     for k, v in service_map.items():
       services_text += f"{k} - {v}\n"
-    update.message.reply_text("Available services:\n" + services_text)
+    update.message.reply_text("Example available services:\n" + services_text)
```
Comment (sonac) [inline burgerbot.py:114]:
```suggestion
    update.message.reply_text("Example of available services:\n" + services_text)
```

**Review by sonac — COMMENTED** [review]
thanks! was thinking about adding smth like this myself, i'll test it on my dev server, cause I have some doubts about formatting of the message (how it will look like in the telegram itself)

File: burgerbot.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def __services(self, update: Update, _: CallbackContext) -> None:
     services_text = ""
     for k, v in service_map.items():
       services_text += f"{k} - {v}\n"
-    update.message.reply_text("Available services:\n" + services_text)
+    update.message.reply_text("Example of available services:\n" + services_text)
```
Comment (harrybiddle) [inline burgerbot.py:114]:
```suggestion
    update.message.reply_text("Available services:\n" + services_text)
```

File: burgerbot.py [hunk 3]
```diff
@@ -120,7 +120,8 @@ def __help(self, update: Update, _: CallbackContext) -> None:
 /stop - stop the bot
 /add_service <service_id> - add service to your list
 /remove_service <service_id> - remove service from your list
-/services - list of available services
+/my_services - view services on your list
+/services - non-exhaustive list of available services
```
Comment (harrybiddle) [inline burgerbot.py:124]:
```suggestion
/services - list of available services
```

**Review by sonac — CHANGES_REQUESTED** [review]
All seems good, you're just missing the handler assignment in bot init (like https://github.com/sonac/burgerbot/blob/master/burgerbot.py#L71)

**Comment by harrybiddle** [comment]
Aha, thanks. I've added that

**Pull request merged**